pkg/scaling/scale_handler.go

```diff
@@ -6,21 +6,20 @@ import (
 	"sync"
 	"time"
 
-	"github.com/kedacore/keda/v2/pkg/eventreason"
-	"k8s.io/client-go/tools/record"
-
 	"github.com/go-logr/logr"
+	kedav1alpha1 "github.com/kedacore/keda/v2/api/v1alpha1"
+	"github.com/kedacore/keda/v2/pkg/eventreason"
+	"github.com/kedacore/keda/v2/pkg/scalers"
+	"github.com/kedacore/keda/v2/pkg/scaling/executor"
+	"github.com/kedacore/keda/v2/pkg/scaling/resolver"
 	"k8s.io/api/autoscaling/v2beta2"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/scale"
+	"k8s.io/client-go/tools/record"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
-
-	kedav1alpha1 "github.com/kedacore/keda/v2/api/v1alpha1"
-	"github.com/kedacore/keda/v2/pkg/scalers"
-	"github.com/kedacore/keda/v2/pkg/scaling/executor"
-	"github.com/kedacore/keda/v2/pkg/scaling/resolver"
```

Comment:
I'd actually prefer if we keep the `github.com/kedacore/keda` related imports in the separate section below general imports.  It's just my ocd, sorry 😄